test/utils/image/manifest.go

```diff
@@ -71,7 +70,6 @@ func initReg() RegistryList {
 	registry := RegistryList{
 		GcAuthenticatedRegistry: "gcr.io/authenticated-image-pulling",
 		E2eRegistry:             "gcr.io/kubernetes-e2e-test-images",
-		E2eVolumeRegistry:       "gcr.io/kubernetes-e2e-test-images/volume",
```

Comment:
Just want to confirm this isn't showing up in any manifests

Reply:
https://github.com/search?q=org%3Akubernetes+E2eVolumeRegistry&type=code
Not found in code or image lists in k/k.

Is there anywhere that I need to check?

Reply:
not too bad ... https://grep.app/search?q=E2eVolumeRegistry and https://cs.k8s.io/?q=E2eVolumeRegistry&i=nope&files=&excludeFiles=&repos=